docs/policies/security_incident_management_policy.md

```diff
@@ -0,0 +1,243 @@
+This policy is applicable to all DfE employees and third-party staff (for example, consultants, resource company employees, temporary employees) that use or have access to DfE systems and / or information. 
+New services in scope for SbD implementation must build and maintain capability to manage Priority 4 (P4) security incidents internally, at the portfolio and service level. This will be referred to as ‘in house’ throughout this document.
+
+## Definition
+
+The Government Functional Standard GovS 007: Security defines a security incident as:
+‘Any circumstance that has arisen contrary to policy and that has the potential to compromise His Majesty’s Government (HMG) assets. Assets include people, property, or information. The circumstance may include actions that were actual or suspected; accidental, deliberate or attempted. “Compromise” is to bring the asset into disrepute or danger’.
+
+An incident in the workplace is an unplanned event that could result in injury, could cause damage to property/departmental assets/information and poses significant risk to merit action and recording. 
+It can also refer to any event that compromises or threatens the confidentiality, integrity, or availability (CIA) of an organisation’s assets. Incidents can include a wide range of events, encompassing physical, personnel, cyber and technical security.
+Managing and responding to incidents is a critical aspect of protective security to prevent or minimise potential damage and protect sensitive information and resources.
+
+## Responsibility
+
+The Chief Information Security Officer (CISO) must be ultimately responsible for the management of all information security incidents.
+
+The Senior Security Advisor (SSA) must be ultimately responsible for the management and resolution of all physical security and personnel security incidents.
+The Cyber and Information Security Division (CISD) Security Incident Management (SIM) team must ensure all incidents are assessed as soon as possible so that a priority can be assigned and the most appropriate course of action can be given for their resolution. The response to an incident is likely to require the skill and expertise of various groups within DfE (for example, IT, operations, legal and human resources) as well as external agencies (for example, police authority and forensic specialists).
+
+The analysis must include the following information:
+
+* identification of type of incident
+* assessment of impact
+* assessment of the scale of the incident
+* identification of classification or type of incident
+
+New services in scope for SbD implementation must manage security incidents ‘in house’ and portfolios and services who choose to manage security incidents ‘in house’ must report any incident through the IT Help Centre immediately on incident identification. This is so the SIM team are aware and have oversight of the incident in the event escalation or further support is required.
```

Comment:
Looks like we're using magic quotes around "in house" e.g. `‘` instead of `"` or `'` - not sure if that matters or not? Just something I noticed

Reply:
Good spot, changed throughout